src/components/FAQ/index.jsx

```diff
@@ -0,0 +1,154 @@
+  },
+  subtitle: {
+    textAlign: 'center',
+  },
+  section: {
+    display: 'flex',
+    flexDirection: 'column',
+    width: '60vw',
+    padding: '1rem',
+    '& .MuiAccordion-root': {
+      marginTop: '.8rem',
+      borderRadius: '10px',
+    },
+    '& .MuiAccordion-root:before': {
+      position: 'inherit',
+    },
+    '& .MuiAccordion-root.Mui-expanded': {
+      margin: '1rem 0 0 0',
+    },
+    '& .MuiAccordionSummary-content': {
+      padding: '0 0.8rem 0 0.8rem',
+      margin: 0,
+    },
+    '& .MuiAccordionDetails-root': {
+      padding: '0 1.8rem 0 1.8rem',
+      display: 'flex',
+      flexDirection: 'column',
+    },
+
+    [theme.breakpoints.down('xs')]: {
+      width: '90vw',
+    },
+  },
+  section__title: {
+    color: theme.palette.tertiary.dark,
+    textAlign: 'start',
+  },
+  section__icon: {
+    color: theme.palette.primary.main,
+  },
+}))
+
+const FAQ = () => {
+  const classes = useStyles()
+  const FAQState = {}
+  DUMMY_QA.qa
+    .map((el) => el.FAQs)
+    .flat()
+    .map((el) => el.id)
+    .forEach((el) => (FAQState[el] = false))
+
+  const [expandedState, setExpandedState] = useState(FAQState)
+
+  const expandHandler = (id) => () => {
+    const newList = { ...expandedState }
+    if (expandedState[id]) {
+      newList[id] = false
+    } else {
+      newList[id] = true
+    }
+    setExpandedState(newList)
+  }
+
+  return (
+    <Box className={classes.root}>
+      <Typography variant="h5" className={classes.title}>
+        Frequently Asked Questions
+      </Typography>
+      <Typography variant="subtitle2" className={classes.subtitle}>
+        Refer to our commonly asked Q&As here!
+      </Typography>
+      {DUMMY_QA.qa.map((qa) => {
+        return (
+          <Box key={`${qa.id}`} className={classes.section}>
+            <Typography variant="h6" className={classes.section__title}>
+              {qa.section}
+            </Typography>
+            {qa.FAQs.map((faq) => {
+              return (
+                <Accordion key={`${faq.id}`} square>
+                  <AccordionSummary
+                    onClick={expandHandler(faq.id)}
+                    expandIcon={
+                      expandedState[faq.id] ? (
+                        <RemoveCircleIcon
+                          className={classes.section__icon}
+                          fontSize="large"
+                        />
+                      ) : (
+                        <AddCircleIcon
+                          className={classes.section__icon}
+                          fontSize="large"
+                        />
+                      )
+                    }
+                  >
+                    <Typography>
+                      <b>{faq.Q}</b>
+                    </Typography>
+                  </AccordionSummary>
+                  <AccordionDetails>
+                    {faq.A.split('\n').map((para, idx) => {
+                      const strToBold = 'nusfintech.ops@gmail.com'
```

Comment:
Hey @Zxun2 this current implementation will work for most of the simple FAQs. However, in the event where there are requests to bold multiple words or include line breaks, this might not be sufficient.

Can we try experimenting usage of a HTML to text approach which might be more scalable? Our answers will be written in HTML eg. `<p>This is the <h3>answer</h3></p>`, which will then be rendered as styled strings. This might be achievable with dangerouslySetInnerHTML or other npm libraries.

https://www.npmjs.com/package/html-react-parser
https://www.npmjs.com/package/html-to-text
https://dev.to/jobpick/how-to-render-html-string-in-a-react-component-3kd2

Reply:
Interesting, I'll give it a try! Some things I'd like to clarify. 

Would inline-html look something like this?

For questions:  `<p> Questions that I would like to ask <p>` or would it be any of the heading tags? `<h1><h1/>` etc

For answers:  `<p>Hello <b>World<b></p><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim.<p>` In this case, the line break is for each `<p></p>` tag.

Also, would there be any inline attributes in the raw HTML data that I need to account for? `<p id="main" class="main">Hello World</p>`

Thank you!